ibis/backends/tests/test_register.py

```diff
@@ -390,10 +394,12 @@ def test_csv_reregister_schema(con, tmp_path):
         "clickhouse",
         "dask",
         "datafusion",
-        "flink",
+        "druid",
+        "exasol" "flink",
```

Comment:
You're missing a comma after `"exasol"`.

Python string literals are automatically concatenated.

```suggestion
        "exasol",
        "flink",
```